src/collective/volto/formsupport/restapi/services/submit_form/field.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+from typing import List, Optional, Any
+
+
+@dataclass
+class Field:
```

Comment:
This represents a value not a field right? it's not the schema of a field definition? The naming should make that clear